db/seeds/initial_seed.rb

```diff
@@ -17,7 +18,7 @@
   { provider_name: "UCL Institute of Education", cip: ucl_cip },
 ].each do |seed|
   provider = LeadProvider.find_or_create_by!(name: seed[:provider_name])
-  provider.update!(cohorts: [cohort_2021]) unless provider.cohorts.any?
+  provider.update!(cohorts: [cohort_2021, cohort_2022]) unless provider.cohorts.any?
```

Comment:
This won't create the cohorts on production so you'll want to make sure they exist before flipping on the feature flag. I'd hold off until later though, as there might be unintended consequences of having multiple cohorts.

Reply:
Yes, my understanding is that we need to manually create both the cohort and the feature flag on production before we flip it on.